src/install/lockfile.zig

```diff
@@ -813,29 +845,32 @@ const Cloner = struct {
             std.mem.set(Tree, trees[1..], Tree{});
         }
 
-        var builder = Tree.Builder{
+        const builder = &Tree.Builder{
```

Comment:
Can you split this into two statements? This looks like it should be a compiler error

Reply:
Do you mean like how it was previously?
```zig
var builder = Tree.Builder{...};
var builder_ = &builder;
```